### PR TITLE
added a new renderer which uses Datatables from datatables.net

### PR DESCRIPTION
**Pull request by tebeso**

### CODE REVIEW

**Comment by tebeso** [comment]
had to recreate this pull request due to a weird error in github

File: src/main/php/PHPMD/Renderer/DatatablesRenderer/DatatablesRenderer.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        }
+        else {
+            // Open this row
+            row.child( format(row.data()) ).show();
+            tr.addClass('shown');
+        }
+    } );
+    
+    
+} );
```
Comment (MarkVaughn) [inline src/main/php/PHPMD/Renderer/DatatablesRenderer/DatatablesRenderer.js:41]:
```suggestion
$(document).ready(function () {
    function format(d) {
        return d[6];
    }

    var table = $('#phpmdDatatable').DataTable({
        columnDefs: [
            {
                targets: 0,
                className: 'details-control',
                orderable: false,
                data: null,
                defaultContent: ''
            },
            {
                targets: [6],
                visible: false
            }
        ],
        pageLength: 100
    });

    $('#phpmdDatatable tbody').on('click', 'td.details-control', function () {
        var tr = $(this).closest('tr');
        var row = table.row(tr);

        if (row.child.isShown()) {
            // This row is already open - close it
            row.child.hide();
            tr.removeClass('shown');
            return;
        }
        // Open this row
        row.child(format(row.data())).show();
        tr.addClass('shown');
    });
});
```

**Comment by tebeso** [comment]
so anything required from my side at this point or whats the status?

**Comment by tebeso** [comment]
anything required here from my side?

**Pull request closed without merging**